docs/install/harvester-configuration.md

```diff
@@ -540,6 +540,21 @@ install:
   persistent_partition_size: 150Gi
 ```
 
+### `install.skipchecks`
+
+#### Definition
+
+The installer automatically checks if the hardware meets the [minimum requirements](./requirements/#hardware-requirements) for production use. If any of the checks fail, installation is stopped, and warnings are printed to the system console and saved to `/var/log/console.log` in the installation environment.
```

Comment:
like https://docs.harvesterhci.io/v1.3/install/harvester-configuration#installforce_efi and others, please specify the default value.

And, a link to PXE example will be more user-friendly.